### PR TITLE
requirePkce setting was flipped

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

File: src/oauth2-client/service.ts [hunk 1]
```diff
@@ -148,7 +148,7 @@ export async function edit(client: OAuth2Client, redirectUris: string[]): Promis
 
   const params: Partial<OAuth2ClientRecord> = {
     allowed_grant_types: client.allowedGrantTypes.join(' '),
-    require_pkce: client.requirePkce?0:1,
+    require_pkce: client.requirePkce?1:0,
```
Comment (pschwyter) [inline src/oauth2-client/service.ts:151]: Here, you dropped these: `  `

Comment (pschwyter) [reply src/oauth2-client/service.ts:151]: not even a chuckle

Comment (evert) [reply src/oauth2-client/service.ts:151]: i just chuckled

Comment (evert) [reply src/oauth2-client/service.ts:151]: SORRY

**Pull request merged**